static/js/composebox_typeahead.js

```diff
@@ -88,6 +88,35 @@ var nextFocus = false;
 function handle_keydown(e) {
     var code = e.keyCode || e.which;
 
+    if ((code === 66 || code === 73 || code === 75) && (e.ctrlKey || e.metaKey)) {
+        var textarea = $("#new_message_content");
+        var range = textarea.range();
+        function add_markdown(markdown) {
+        if (!document.execCommand('insertText', false, markdown)) {
+            textarea.range(range.start, range.end).range(markdown);
+            }
+        }
+
+        if (range.length > 0) {
+            if (code === 66) {
+                // convert selected text to bold text
+                add_markdown("**"+range.text+"**");
+            }
+            if (code === 73) {
+                // convert selected text to italic text
+                add_markdown("*"+range.text+"*");
+            }
+            if (code === 75) {
```

Comment:
Why ctrl+K?

Reply:
I think that's the default for links. Don't remember from where.. I think GitHub maybe.

Reply:
Yeah, in GitHub and also ctrl+K is used for inserting link in most text writers. We can change it according to user convenience, though.